src/pretix/control/forms/vouchers.py

```diff
@@ -332,6 +335,12 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._set_field_placeholders('send_subject', ['event', 'name'])
         self._set_field_placeholders('send_message', ['event', 'voucher_list', 'name'])
+
+        with language(self.instance.event.settings.locale, self.instance.event.settings.region):
+            for f in ("send_subject", "send_message"):
+                print(repr(self.fields[f].initial), self.instance.event.settings.locale)
+                self.fields[f].initial = str(_(self.fields[f].initial))
```

Comment:
See above, we do not need `gettext_noop`, that actual solution here is the conversion to `str()` which fixes the translation into a string with the event’s locale in the with-statement.

```suggestion
                self.fields[f].initial = str(self.fields[f].initial)
```